app/controllers/RegularContributions.scala

```diff
@@ -70,32 +71,74 @@ class RegularContributions(
       )
     }
 
-  def status(jobId: String): Action[AnyContent] = authenticatedAction().async { implicit request =>
+  def displayFormMaybeAuthenticated(): Action[AnyContent] =
+    maybeAuthenticatedAction(recurringIdentityClientId).async { implicit request =>
+      request.user.fold {
+        val uatMode = testUsers.isTestUserOptionalAuth(request)
+        Future(
```

Comment:
I think using Future.successful is more efficient

Reply:
👍